swagger.yaml

```diff
@@ -5672,14 +5673,8 @@ definitions:
         type: string
         format: date-time
         x-go-custom-tag: gorm:"type:timestamp with time zone"
-      node_updater_started_at:
```

Comment:
Why was this removed?

Reply:
I created it at the start of the epic.  Decided that I don't need it.